### DISCUSSION TITLE
Get List of all contributors registered

### DISCUSSION THREAD

**Discussion by sachinsahu731** [Sunbird-coKreat/Community#45] [General Discussions]
We need a list of all  contributors registered on VDN -- Contributor Name, Individual/Org.
Kindly let me know hot to get this .

If we need this only from backend then mention the database name and where we will get in server LOCATION .

**Comment by vaivk369** [comment]
Hi Sachin,

The DB name is opensaberdb
The user entities you will find in V_User table
The Org entities you will find in V_Org
And the mapping of user with the org you will find in V_User_Org table.

**Reply by kameshbhr** [reply]
@vaivk369 Thanks for the details. Does V_User_table only have users registered as contributors? If not, can you please let us know how can we get the users who are registered as individual contributors from this table?
Can you please share if there is any document that details the schema of these tables.

**Reply by vaivk369** [reply]
The User schema stores information about all the contributing Users. User can be an admin of an contributing organization or can be a user of and contributing organization or can be an Individual contributor. If the user is an individual contributor this schema has roles column set as ["individual"], for contributor Org Admin it is set to ["admin"], for and for a contributing Org User it is set to ["User"]. 

Here is the document having details of the schema: https://cokreat.sunbird.org/learn/product-and-developer-guide/asset-sourcing/contribution-registry

**Reply by kameshbhr** [reply]
@vaivk369 Thank you. It would be helpful if you could update the DB name and table names in the schema details in the the document for future reference.
